card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -41,14 +41,16 @@ def wrap_item item, args={}
     _render_core args
   end
 
-  view :edit do |args|
-    super(args.merge(:pointer_item_class=>'form-control'))
-  end
+#  view :edit do |args|
+#    super(args.merge(:pointer_item_class=>'form-control'))
+#  end
```

Comment:
we actually only want the form-control class on controls like input, password, textarea, etc.  We *don't* want it on checkboxes and radios (it was causing a lot of headaches).  this is much cleaner.